### PR TITLE
Add ability for plugins to access ride's chain lift speed

### PR DESCRIPTION
**Pull request by csm10495**
I wanted to use this info in a plugin, but couldn't find it... so added it.

### CODE REVIEW

File: distribution/openrct2.d.ts [hunk 1]
```diff
@@ -1064,6 +1064,21 @@ declare global {
          * The percentage of downtime for this ride from 0 to 100.
          */
         readonly downtime: number;
+
+        /**
+         * The currently set chain lift speed.
+         */
+        lift_hill_speed: number;
```
Comment (Broxzier) [inline distribution/openrct2.d.ts:1071]: Please change this to use camel Case for variables. This also applies to the rest.

File: src/openrct2/scripting/bindings/ride/ScRide.cpp [hunk 2]
```diff
@@ -494,6 +495,35 @@ namespace OpenRCT2::Scripting
         return ride != nullptr ? ride->downtime : 0;
     }
 
+    uint8_t ScRide::lift_hill_speed_get() const
```
Comment (Broxzier) [inline src/openrct2/scripting/bindings/ride/ScRide.cpp:498]:
```suggestion
    uint8_t ScRide::liftHillSpeed_get() const
```

**Review by Broxzier — CHANGES_REQUESTED** [review]

**Comment by csm10495** [comment]
@Broxzier I updated the PR thanks.

**Review by Broxzier — COMMENTED** [review]
I've rebased this PR, added a changelog entry and incremented the API version. Found one more thing to improve on, since the numbers did not match when the game was set to use the metric system.

**Comment by Broxzier** [comment]
@csm10495 I noticed that you worked directly on the `develop` branch for this PR. While that works, it makes it more difficult for you to manage your branches, especially now that it has been merged. It's best if you reset your branch to match the OpenRCT2 repository, and that you work on feature branches for future PRs.

**Pull request merged**